dsl/src/main/java/tech/pegasys/peeps/node/Web3Provider.java

```diff
@@ -65,8 +66,9 @@
 
   public Web3Provider(final Web3ProviderConfiguration config, final GenericContainer<?> container) {
     this.container = container;
-    this.nodeRpc = new NodeRpcClient(config.getVertx(), dockerLogs());
-    this.rpc = new NodeRpcMandatoryResponse(nodeRpc);
+    this.signerRpcClient =
```

Comment:
Why does this need a signerRpcClient?

Reply:
Really just upgraded this from a NodeRpcCLient, such that we can do signining things (yes, this will fail if called on a Besu node)